environment.yml

```diff
@@ -1,34 +1,34 @@
 # You can use this file to create a conda environment for this pipeline:
 #   conda env create -f environment.yml
-name: nf-core-rnaseq-1.4dev
+name: nf-core-rnaseq-1.4
 channels:
   - conda-forge
   - bioconda
   - defaults
 dependencies:
   ## conda-forge packages
   - r-base=3.5.1
-  - conda-forge::r-data.table=1.12.2
+  - conda-forge::r-data.table=1.12.4
   - conda-forge::r-gplots=3.0.1.1
-  - conda-forge::r-markdown=0.9
+  - conda-forge::r-markdown=1.1
   - matplotlib=3.0.3        # Current 3.1.0 build incompatible with multiqc=1.7
 
   ## bioconda packages
   - sortmerna=2.1b # for metatranscriptomics
   - fastqc=0.11.8
-  - trim-galore=0.6.2
+  - trim-galore=0.6.4
   - star=2.6.1d             # Don't upgrade me - 2.7X indices incompatible with iGenomes.
   - rsem=1.3.2
-  - salmon=0.14.0
+  - salmon=0.14.2
```

Comment:
all these package updates need to be added to changelog?

Reply:
Indeed - will do that now quickly 👍

Reply:
Also sorted the list, added a comment that we dropped `csvtk` from the environment. 

Thanks for spotting these!